shared/js/smart-screen/scrollable.js

```diff
@@ -162,6 +162,11 @@
       this._setScrollStyle();
     },
 
+    scrollToFirstItem: function() {
```

Comment:
Why do we need to set `translateX` back to 0?

And I would prefer adding `getItem()` and use `scrollable.scrollTo(scrollable.getItem(0))` if we really need it.